source/replication/site-replication-overview.rst

```diff
@@ -0,0 +1,541 @@
+
+.. admonition:: Wait Until Replication Completes
+   :class: warning
+
+   If there is existing data on the first peer site, wait until :mc-cmd:`mc admin replicate status` shows that the replication process has completed before performing read or write operations on the other peer sites. 
+
+
+After the initial replication completes, changes to :ref:`items that replicate <minio-site-replication-what-replicates>` on any of the peer sites synchronize to all of the other peer sites.
+Changes include creating, modifying, or deleting such items.
+
+All Sites Must Use the Same Root Deployment Credentials
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+All sites must have the same root user credentials (for example, ``MINIO_ROOT_USER``, ``MINIO_ROOT_PASSWORD``).
+This allows MinIO to log in across each site.
+
+All Sites Must Use the Same IDP
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+All sites must use the same :ref:`Identity Provider <minio-authentication-and-identity-management>`.
+Site replication supports the included MinIO IDP, OIDC, or LDAP.
+
+Access to the Same Encryption Service
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For :ref:`SSE-S3 <minio-encryption-sse-s3>` or :ref:`SSE-KMS <minio-encryption-sse-kms>` encryption via Key Management Service (KMS), all sites must have access to a central KMS deployment. 
+
+You can achieve this with a central KES server or multiple KES servers (say one per site) connected via a central supported :ref:`key vault server <minio-sse>`.
+
+Tutorials
+---------
+
+Configure Site Replication
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. tab-set::
+
+   .. tab-item:: Console
+
+      #. :ref:`Deploy <deploy-minio-distributed>` two or more separate MinIO sites, using the same Identity Provider for each site
+
+         Only one site can have any buckets or objects on it.
+         The other site(s) must be empty.
+
+      #. In a browser, access the Console for the site with data (if any)
+
+         For example, ``https://<addressforsite>:9000``
+         
+         Replace ``<addressforsite>`` with the IP address or URL for the MinIO deployment.
+
+      #. Select **Settings**, then **Site Replication**
+
+         .. image:: /images/minio-console/console-settings-site-replication.png
+            :width: 400px
+            :alt: MinIO Console menu with the Settings heading expanded to show Site Repilication
+            :align: center
+      
+      #. Select :guilabel:`Add Sites +`
+
+         .. image:: /images/minio-console/console-settings-site-replication-add.png
+            :width: 600px
+            :alt: MinIO Console's Add Sites for Replication screen
+            :align: center
+
+      #. Complete the requested information for the site:
+
+         :Access Key: `(required)` The user name for ``root`` to use for signing in to each site. Must be the same across all sites.
+
+         :Secret Key: `(required)` The password for ``root`` to use for signing in to each site. Must be the same across all sites.
```

Comment:
root access key and secret key need not be identical across sites.

Reply:
Removed.